### DISCUSSION TITLE
Notification error

### DISCUSSION THREAD

**Discussion by siarheipashkevich** [spatie/laravel-backup#1790] [Q&A]
Hi, I'm setting the notifications
<img width="1021" alt="image" src="https://github.com/spatie/laravel-backup/assets/7753600/2902a9b9-fa93-497f-9365-2970588dcc37">

But I'm getting errors in the slack and sentry
![image](https://github.com/spatie/laravel-backup/assets/7753600/e3e9f5a0-2854-49e4-8fd5-cbd175e133db)
<img width="1563" alt="image" src="https://github.com/spatie/laravel-backup/assets/7753600/5cc8b01e-979e-4ec9-ba08-ea3ce80c321c">

**Comment by mr-wolf-gb** [comment]
in my case the exception message : Driver [slack] not supported.
